src/Nest/CommonOptions/Geo/Distance.cs

```diff
@@ -7,7 +7,7 @@
 namespace Nest
 {
 	[JsonFormatter(typeof(DistanceFormatter))]
-	public class Distance
+	public class Distance : IFormattable
```

Comment:
`IFormattable` shouldn't be implemented; the string representation of `Distance` should always use `CultureInfo.InvariantCulture`

Reply:
Makes sense if string representation is only used for serialization, which I assume it is.